lms/static/scripts/frontend_apps/utils/vitalsource-sample-data.js

```diff
@@ -0,0 +1,345 @@
+      cfi: '/6/52[;vnd.vst.idref=vst-a6c7b7bf-7e6b-4aab-829f-ab61fbd7d367]',
+      page: '26',
+    },
+    {
+      title: 'Learning Check: Creating Highlights and Managing Highlighters',
+      cfi: '/6/54[;vnd.vst.idref=vst-47f9ee6e-042a-46fc-8eee-daeb5f9b4fd8]',
+      page: '27',
+    },
+    {
+      title: 'Subscribe to Other Bookshelf Users',
+      cfi: '/6/56[;vnd.vst.idref=vst-571fdb7c-2547-4bed-8bc6-59821e36092b]',
+      page: '28',
+    },
+    {
+      title: 'Share Highlights and Notes',
+      cfi: '/6/58[;vnd.vst.idref=vst-eaf11d17-7b35-4189-82d9-299a35aeab78]',
+      page: '29',
+    },
+    {
+      title: 'Video Demonstration: Sharing and Subscribing',
+      cfi: '/6/60[;vnd.vst.idref=vst-e46a628b-ab7c-42f9-8e7a-5aafd1503001]',
+      page: '30',
+    },
+    {
+      title: 'Manage Your Notebook',
+      cfi: '/6/62[;vnd.vst.idref=vst-6d6266f0-4b68-4b7d-a0e0-1322c62e6389]',
+      page: '31',
+    },
+    {
+      title: 'Study with Review Mode',
+      cfi: '/6/64[;vnd.vst.idref=vst-af780b01-1961-42a7-8aa5-8c0da20aaa04]',
+      page: '32',
+    },
+    {
+      title: 'Video Demonstration: Notebook and Review Mode',
+      cfi: '/6/66[;vnd.vst.idref=vst-aca7b9db-ba85-4393-87a2-fb7ad6b4c3c4]',
+      page: '33',
+    },
+    {
+      title: 'Learning Check: Notebook and Review Mode',
+      cfi: '/6/68[;vnd.vst.idref=vst-76d16952-eb1b-400e-92b6-7d18b656afa1]',
+      page: '34',
+    },
+    {
+      title: 'Study with Flash Cards',
+      cfi: '/6/70[;vnd.vst.idref=vst-2b3481e1-596a-49ac-a50b-76ea73658c18]',
+      page: '35',
+    },
+    {
+      title: 'Video Demonstration: Creating Flashcards',
+      cfi: '/6/72[;vnd.vst.idref=vst-b19a9bba-1155-4944-82ef-73d1e71891ea]',
+      page: '36',
+    },
+    {
+      title: 'Learn with Text-to-Speech',
+      cfi: '/6/74[;vnd.vst.idref=vst-65cd738c-65d4-486e-9eee-726de86c85db]',
+      page: '37',
+    },
+    {
+      title: 'Part 5: Recap and Action Items',
+      cfi: '/6/76[;vnd.vst.idref=vst-2ee70751-8a10-432a-8918-1c49810add84]',
+      page: '38',
+    },
+    {
+      title: 'Steps to Make Bookshelf a Part of Your Routine',
+      cfi: '/6/76[;vnd.vst.idref=vst-2ee70751-8a10-432a-8918-1c49810add84]',
+      page: '38',
+    },
+    {
+      title: 'VitalSource Support Channels',
+      cfi: '/6/78[;vnd.vst.idref=vst-b924e0ab-57dd-4aaa-b9d6-3012add8f2d8]',
+      page: '39',
+    },
+  ],
+
+  9781400847402: [
+    {
+      title: 'Cover Page',
+      cfi: '/6/2[;vnd.vst.idref=cover]',
+      page: 'i',
+    },
+    {
+      title: 'Title Page',
+      cfi: '/6/6[;vnd.vst.idref=title]',
+      page: 'iii',
+    },
+    {
+      title: 'Copyright Page',
+      cfi: '/6/8[;vnd.vst.idref=copy]',
+      page: 'iv',
+    },
+    {
+      title: 'Dedication Page',
+      cfi: '/6/10[;vnd.vst.idref=ded]',
+      page: 'v',
+    },
+    {
+      title: 'Contents',
+      cfi: '/6/12[;vnd.vst.idref=toc]',
+      page: 'vii',
+    },
+    {
+      title: 'Forward',
```

Comment:
Super pedantic, but, hey, it totally caught my eye when tootling around the interface: this should be _Foreword_. It was likely misspelled in your source data, though?

Reply:
Yes. I took the source data verbatim. I spotted various other apparent errors in the data as well, such as duplicated chapters. This might be something we have to somehow deal with on the server-side by filtering it before serving it to the client.